On Debian systems, we want libdrizzle-1.0 headers.--- a/percona_playback/libdrizzle_client/libdrizzle_client.cc
--- a/percona_playback/libdrizzle_client/libdrizzle_client.cc
+++ b/percona_playback/libdrizzle_client/libdrizzle_client.cc
@@ -24,8 +24,8 @@
 
 
 #include <percona_playback/db_thread.h>
-#include <libdrizzle/drizzle.h>
-#include <libdrizzle/drizzle_client.h>
+#include <libdrizzle-1.0/drizzle.h>
+#include <libdrizzle-1.0/drizzle_client.h>
 
 class QueryResult;
 class LibDrizzleOptions;
--- a/percona_playback/tcpdump/connection_state.cc
+++ b/percona_playback/tcpdump/connection_state.cc
@@ -14,8 +14,8 @@
  * END LICENSE */
 #include <pcap.h>
 #include <iostream>
-#include <libdrizzle/drizzle_server.h>
-#include <libdrizzle/row_client.h>
+#include <libdrizzle-1.0/drizzle_server.h>
+#include <libdrizzle-1.0/row_client.h>
 #include <stdio.h>
 #include <stdint.h>
 #include <assert.h>
--- a/percona_playback/tcpdump/connection_state.h
+++ b/percona_playback/tcpdump/connection_state.h
@@ -22,7 +22,7 @@
 #include <string>
 #include <string.h>
 #include <boost/shared_ptr.hpp>
-#include <libdrizzle/drizzle_client.h>
+#include <libdrizzle-1.0/drizzle_client.h>
 
 #include "percona_playback/query_result.h"
 #include "percona_playback/db_thread.h"
